packages/web/src/scss/components/FileUploader/README.md

```diff
@@ -219,11 +219,11 @@ or `is-disabled` to the `FileUploaderInput` subcomponent as well.
 </div>
 ```
 
-#### JavaScript-Controlled Validation Message
+#### JavaScript-Controlled Validation Text
 
 When implementing client-side form validation, use JS interaction state classes
 (`has-success`, `has-warning`, `has-danger`) on the wrapping `<div>` element and
-render validation messages in a `<div>` with `data-spirit-element="validator_message"`
+render validation texts in a `<div>` with `data-spirit-element="validator_message"`
```

Comment:
@adamkudrna @crishpeen What about the validator_message? I think we do not have an issue with this. Or do we want to change this data element value?